net.brinkervii.grapejuice.yml

```diff
@@ -0,0 +1,122 @@
+app-id: net.brinkervii.grapejuice 
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: grapejuice
+separate-locales: false
+
+finish-args:
+        - --share=network
+        - --socket=x11
+        - --share=ipc
+        - --socket=wayland
+        - --device=dri
+        - --allow=devel
+        - --socket=pulseaudio
+        - --allow=multiarch
+
+add-extensions:
+        org.freedesktop.Platform.Compat.i386:
+                directory: lib/i386-linux-gnu
+                version: '21.08'
+        org.freedesktop.Platform.Compat.i386.Debug:
```

Comment:
I don't think you need/want this

Reply:
This is required, since wine won't run without multilib support

Reply:
`org.freedesktop.Platform.Compat.i386.Debug` is the debug package. ie it contain the debug symbols, nothing else.

Reply:
Makes sense, I will remove those